SPECS/kubernetes/kubernetes-1.17.11.spec

```diff
@@ -80,13 +84,17 @@ for component in ${components_to_build}; do
 done
 
 %check
-cd %{_builddir}/%{name}/src
-components_to_test=$(ls -1 %{_builddir}/%{name}/node/bin)
+# patch test script so it supports golang 1.15 which is now used to build kubernetes
+cd %{_builddir}/%{name}/src/hack/make-rules
+patch -p1 test.sh < %{SOURCE2}
```

Comment:
Why not use the `%(auto)setup` to patch?

Reply:
can't use that see comment above